src/kOS/Binding/BindingsUniverse.cs

```diff
@@ -6,16 +6,21 @@
 using System.Collections.Generic;
 using UnityEngine;
 using kOS.Safe.Persistence;
+using kOS.Communication;
 
 namespace kOS.Binding
 {
     [Binding("ksp")]
     public class BindingTimeWarp : Binding
     {
+        private HomeConnection homeConnection;
+        private ControlConnection controlConnection;
+
         public override void AddTo(SharedObjects shared)
         {
             shared.BindingMgr.AddGetter("KUNIVERSE", () => new KUniverseValue(shared));
-
+            shared.BindingMgr.AddGetter("HOMECONNECTION", () => homeConnection ?? (homeConnection = new HomeConnection(shared)));
+            shared.BindingMgr.AddGetter("CONTROLCONNECTION", () => controlConnection ?? (controlConnection = new ControlConnection(shared)));
```

Comment:
So are these meant to be inter-vessel connections you can send messages on in scripts?  I guess I'll have to wait until I get to the later commit where the docs are.